index.html

```diff
@@ -22,7 +22,7 @@
     <link rel="icon" type="image/png" sizes="16x16" href="/public/icons/favicon-16x16.png">
 
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Portfolio</title>
+    <title>Raia</title>
```

Comment:
Update the title tag to reflect the correct page title for better user experience and SEO. 

Suggested change:
```html
<title>Raia</title>
``` 

For more information, see [MDN HTML Title Element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/title).